gallery/src/pages/StylingComponents.tsx

```diff
@@ -85,6 +85,48 @@ const StylingRSCPage = () =>
         visual state. These classes are common across all components that might need them, especially in the case of the
         validation utility classes.
       </p>
+
+      <h3 className="nx-h3 nx-tile__section-header">CSS Resources</h3>
```

Comment:
Shouldn't this be in a separate `.nx-tile__subsection`?

Reply:
> Shouldn't this be in a separate `.nx-tile__subsection`?

Yup: https://github.com/sonatype/sonatype-react-shared-components/pull/308/commits/d92adcbbde5c560e9dbb4d8ce989930c2219fcb6